app/services/cache/ExportsCacheService.scala

```diff
@@ -30,9 +30,10 @@ class ExportsCacheService @Inject()(journeyCacheModelRepo: ExportsCacheModelRepo
   def update(sessionId: String, model: ExportsCacheModel): Future[Option[ExportsCacheModel]] =
     journeyCacheModelRepo.upsert(sessionId, model.copy(updatedDateTime = now()))
 
+  @deprecated("Please use `get` and `ExportCacheModel#itemBy` methods")
   def getItemByIdAndSession(itemId: String, sessionId: String): Future[Option[ExportItem]] =
```

Comment:
Are there any usages?
Cant we just change them?

Reply:
I use `deprecated` in development to find all changes. I will remove this before merge.